plugins/source/azuredevops/resources/services/core/projects.go

```diff
@@ -0,0 +1,46 @@
+package core
+
+import (
+	"context"
+
+	"github.com/cloudquery/cloudquery/plugins/source/azuredevops/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/core"
+)
+
+func Projects() *schema.Table {
+	return &schema.Table{
+		Name:     "azuredevops_core_projects",
+		Resolver: fetchProjects,
+		Columns: []schema.Column{
+			{
+				Name:     "id",
+				Type:     schema.TypeUUID,
+				Resolver: schema.PathResolver("Id"),
+			},
+		},
+	}
+}
+
+func fetchProjects(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	cl := meta.(*client.Client)
+	coreClient, err := core.NewClient(ctx, cl.Connection)
+	if err != nil {
+		return err
+	}
+
+	stateFilter := &core.ProjectStateValues.All
+	projects, err := coreClient.GetProjects(ctx, core.GetProjectsArgs{StateFilter: stateFilter})
```

Comment:
```go
input := core.GetProjectsArgs{StateFilter: &core.ProjectStateValues.All}
for {
	projects, err := coreClient.GetProjects
	if err != nil {
		return err
	}

	res <- projects.Value

	if len(projects.ContinuationToken) == 0 {
		break
	}

	input.ContinuationToken = &projects.ContinuationToken
}
```

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/5611/commits/5434df748cdaf9964d9597e10985da2fd509c8a8